frontend/src/component/project/Project/ProjectInsights/ProjectInsightsStats/ProjectInsightsStats.tsx

```diff
@@ -0,0 +1,136 @@
+import { Box, styled, Typography } from '@mui/material';
+import type { ProjectStatsSchema } from 'openapi/models';
+import { HelpPopper } from './HelpPopper';
+import { StatusBox } from './StatusBox';
+
+const StyledBox = styled(Box)(({ theme }) => ({
+    display: 'grid',
+    gap: theme.spacing(2),
+    gridTemplateColumns: 'repeat(5, 1fr)',
+    flexWrap: 'wrap',
+    [theme.breakpoints.down('lg')]: {
+        gridTemplateColumns: 'repeat(2, 1fr)',
+    },
+    [theme.breakpoints.down('sm')]: {
+        flexDirection: 'column',
+    },
+}));
+
+const StyledWidget = styled(Box)(({ theme }) => ({
+    position: 'relative',
+    padding: theme.spacing(3),
+    backgroundColor: theme.palette.background.paper,
+    flex: 1,
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    justifyContent: 'center',
+    borderRadius: `${theme.shape.borderRadiusLarge}px`,
+    [theme.breakpoints.down('lg')]: {
+        padding: theme.spacing(2),
+    },
+}));
+
+const StyledTimeToProductionDescription = styled(Typography)(({ theme }) => ({
+    color: theme.palette.text.secondary,
+    fontSize: theme.typography.body2.fontSize,
+    lineHeight: theme.typography.body2.lineHeight,
+}));
+
+interface IProjectStatsProps {
+    stats: ProjectStatsSchema;
+}
+
+export const ProjectInsightsStats = ({ stats }: IProjectStatsProps) => {
+    if (Object.keys(stats).length === 0) {
+        return null;
+    }
+
+    const {
+        avgTimeToProdCurrentWindow,
+        projectActivityCurrentWindow,
+        projectActivityPastWindow,
+        createdCurrentWindow,
+        createdPastWindow,
+        archivedCurrentWindow,
+        archivedPastWindow,
+    } = stats;
+
+    return (
+        <StyledBox>
+            <StyledWidget>
+                <StatusBox
+                    title='Total changes'
+                    boxText={String(projectActivityCurrentWindow)}
+                    change={
+                        projectActivityCurrentWindow - projectActivityPastWindow
+                    }
+                >
+                    <HelpPopper id='total-changes'>
+                        Sum of all configuration and state modifications in the
+                        project.
+                    </HelpPopper>
+                </StatusBox>
+            </StyledWidget>
+            <StyledWidget>
+                <StatusBox
+                    title='Avg. time to production'
+                    boxText={
+                        <Box
+                            sx={{
```

Comment:
StyledComponent

Reply:
I will make a note to follow up with this in the next iteration